src/engraving/dom/cmd.cpp

```diff
@@ -3273,46 +3275,49 @@ void Score::cmdMirrorNoteHead()
 
 void Score::cmdIncDecDuration(int nSteps, bool stepDotted)
 {
-    EngravingItem* el = selection().element();
-    if (el == 0) {
-        return;
-    }
-    if (el->isNote()) {
-        el = el->parentItem();
-    }
-    if (!el->isChordRest()) {
-        return;
-    }
-
-    ChordRest* cr = toChordRest(el);
-
-    // if measure rest is selected as input, then the correct initialDuration will be the
-    // duration of the measure's time signature, else is just the input state's duration
-    TDuration initialDuration;
-    if (cr->durationType() == DurationType::V_MEASURE) {
-        initialDuration = TDuration(cr->measure()->timesig(), true);
-
-        if (initialDuration.fraction() < cr->measure()->timesig() && nSteps > 0) {
-            // Duration already shortened by truncation; shorten one step less
-            --nSteps;
+    if (selection().isRange()) {
```

Comment:
It seems we can just use `m_selection` throughout this method, to avoid the numerous calls

Reply:
OK, done